holmes/core/llm.py

```diff
@@ -535,6 +536,7 @@ def _create_robusta_model_entry(
         self, model_name: str, args: Optional[dict[str, Any]] = None
     ) -> dict[str, Any]:
         entry = self._create_model_entry(
+            # this also seems like a mistake? (wont token counts be wrong etc)
```

Comment:
@moshemorad and @nherment - same comment here.

Reply:
Same as above.